### ISSUE TITLE
Visual / UX / UI improvements

### ISSUE THREAD

**Issue by Vaalyn** [AzuraCast/AzuraCast#1501]
In order to improve the design UI & UX of AzuraCast I'll be working on a prototype branch where I'll test out some proposed changes to different UI elements. I want to use this issue to aggregate some of these changes so that we have a list of things and maybe also get some feedback and further ideas.

### Improve usage of Material Design
As AzuraCast uses the [Material Design](https://material.io/design/) language for the interface design we already have a really good guideline on how to implement different elements but we violate that in some parts which may impact the look & feel for the user.

Here is a screenshot of an example a user provided me with on what could be done:
![image](https://user-images.githubusercontent.com/13745863/57026180-6ed68900-6c39-11e9-935f-a620255a7ed8.png)

When compared to the current login screen of AzuraCast this example has a nicer flow for the elements which makes it more visually pleasing. Although I'd add some background, probably light dark transparent to that so that the text labels won't become unreadable when the background image is differently positioned on other screen sizes.

ToDo List:

- [ ] Use Material Design for checkboxes in tables
    - Partly Blocked by #1590
    - Media Manager already done
- [ ] Make complete rows clickable for selecting checkboxes
    - Blocked by #1590
- [x] Improve usability / discoverability of the "remove media from playlist" functionality
    - Currently when you want to remove a song from a playlist you need to select it via the checkbox then click on `Set Playlists` and check then uncheck the box of the playlist you want to remove it from.
    - This is not very obvious for new users

![image](https://user-images.githubusercontent.com/13745863/61716194-e6aae000-ad5e-11e9-87d9-7674d116a617.png)

**Comment by SlvrEagle23** [comment]
@Vaalyn I like the way that looks, and I think we can probably make our existing theme setup work a little closer to that, but there's a reason we can't just adopt a theme like the one suggested by the user in question: we don't have (and can't get) a license to distribute it in an open-source piece of software.

One of the big reasons we had a theme switch a little while back was to make sure that we were on a theme that was fully open all the way through, and that wouldn't run us into trouble with licensing. It was one of many components we had to update due to questions of licensing, including our logo (hence the slightly updated one with the little waves in the background). Now everything is 100% open and we have a licensing paper trail for our entire branding.

We should be exceedingly careful when implementing new visual components that we aren't stepping into dangerous waters by implementing things directly from themes that we can't redistribute.

**Comment by Vaalyn** [comment]
The first iteration of improvements is now merged in 3dcc087487c65b544dc444e014e1c052d73cd4f1.

I'd like to get some feedback on the changes from our users. If you have some time please let us hear your opinions about the changes.

**Comment by vollknorke1337** [comment]
Additional improvement:

- use material design for the checkboxes 
- make the whole lne clickable, not only the small checkbox

![image](https://user-images.githubusercontent.com/25418622/58368647-00ab7c00-7ef0-11e9-868d-004f7ba39e16.png)

**Comment by Vaalyn** [comment]
@vollknorke1337 I've tried to implement the checkbox design that we use for all other checkboxes in the bootgrid table there but this doesn't really want to work with it nicely.

I can override the template of the select column through the bootgrid config but bootgrid does not make it possible to obtain a row id in the template for this element which makes it quite hard to assign the custom styled checkbox (which is just a label styled to look like a checkbox) to the input element.

For the moment it's there doesn't seem to be a quick & easy way to implement this with the existing setup for data tables. I'll keep it on my todo list for these improvements though.

**Comment by SlvrEagle23** [comment]
@vollknorke1337 Also, the reason whole-row clicking doesn't work on that section is because sadly with Bootgrid this overrides the click events on any of the individual elements, so if you were to click on the play button to preview the song or the "Edit"/"Rename" buttons on the right those would stop working.

**Comment by vollknorke1337** [comment]
Thanks for your answers. Maybe there is another way to fix this in future, because now it's hard to hit that Boxes in a rush. Just for purposes of comfort. 

Keep the work on - nevertheless I will install AzuraCast soon on my server. :)
